lxd/network/driver_macvlan.go

```diff
@@ -21,6 +24,44 @@ func (n *macvlan) DBType() db.NetworkType {
 	return db.NetworkTypeMacvlan
 }
 
+// State returns the network state.
+func (n *macvlan) State() (*api.NetworkState, error) {
+	parentState, err := resources.GetNetworkState(GetHostDevice(n.config["parent"], n.config["vlan"]))
+	if err != nil {
+		// If the parent is not found, return a response indicating the network is unavailable.
+		if api.StatusErrorCheck(err, http.StatusNotFound) {
+			return &api.NetworkState{
+				State: "unavailable",
+				Type:  "unknown",
+			}, nil
+		}
+
+		// In all other cases, return the original error.
+		return nil, err
+	}
+
+	var mtu int
+
+	configMtu := n.config["mtu"]
+	if configMtu != "" {
+		mtu, err = strconv.Atoi(configMtu)
```

Comment:
```suggestion
		mtu, err := strconv.ParseUint(d.config["mtu"], 10, 32)
```

To get a `uint` and for consistency with other places where we handle the MTU.